llvm/utils/llvm-original-di-preservation.py

```diff
@@ -487,14 +490,15 @@ def Main():
 
                 if bugs_metadata == "DILocation":
                     try:
+                        origin = bug["origin"]
                         action = bug["action"]
                         bb_name = bug["bb-name"]
                         fn_name = bug["fn-name"]
                         instr = bug["instr"]
                     except:
                         skipped_bugs += 1
                         continue
-                    di_loc_bug = DILocBug(action, bb_name, fn_name, instr)
+                    di_loc_bug = DILocBug(origin, action, bb_name, fn_name, instr)
```

Comment:
Maybe move `origin` to the end, which also seems to match the order where its used?

Reply:
Oddly the parameters to this function are passed in reverse of their use/declaration; wasn't sure why, but I decided to keep the ordering rather than make the new parameter an odd one out.

Reply:
Ah hmm, that is odd... Okay, seems like you've tried to follow the local (somewhat strange) style. 😄